azerothcore/azerothcore-wotlk
# fix(DB/Creature):improved Defender of Timbermaw

Fix for Defender of Timbermaw.

On spawn it will cast heal.
While active, it will help with few Lighting Bolts on random enemies.

To test:

.additem 21326
.mod rep 576 50000

Pull few mobs and damage then just a little to have agrro.
Use trinket and wait for heal / damage on them.

Should close #3521
